core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/SQLPlanParser.scala

```diff
@@ -353,10 +353,10 @@ object SQLPlanParser extends Logging {
     val funcName = functionPattern.findFirstMatchIn(expr) match {
       case Some(func) =>
         val func1 = func.group(1)
-        // `cast` is not an expression hence should be ignored. In the physical plan cast is
-        // usually presented as function call for example: `cast(value#9 as date)`. We add
+        // `cast` and `Some` are not expressions hence should be ignored. In the physical plan
+        // cast is usually presented as function call for example: `cast(value#9 as date)`. We add
         // other function names to the result.
-        if (!func1.equalsIgnoreCase("cast")) {
+        if (!func1.equalsIgnoreCase("cast") && !func1.equalsIgnoreCase("Some")) {
```

Comment:
Thanks @amahussein. I created an array so that we can add other functions in the future if required. Updated the array with the functions which are similar to Some. Also updated with the names which I saw after running it on  integration tests eventlogs. PTAL.